src/codegen.cpp

```diff
@@ -3,7 +3,7 @@
 #include "llvm-version.h"
 #include "platform.h"
 #include "options.h"
-#if defined(_OS_WINDOWS_) && JL_LLVM_VERSION < 30900
+#if defined(_OS_WINDOWS_) && JL_LLVM_VERSION < 40000
```

Comment:
@vtjnash might know the answer for that.

My hypothesis is that we were hoping that COFF support might be fixed eventually.